controllers/nginx/pkg/cmd/controller/metrics.go

```diff
@@ -40,7 +42,10 @@ func (em exeMatcher) MatchAndName(nacl common.NameAndCmdline) (bool, string) {
 }
 
 func (n *NGINXController) setupMonitor(args []string) {
-	pc, err := newProcessCollector(true, exeMatcher{"nginx", args})
+	var enableVts = true
```

Comment:
@gianrubio I think [after this line](https://github.com/kubernetes/ingress/blob/master/controllers/nginx/pkg/cmd/controller/nginx.go#L312) is the most appropriate place.